ConsoleApp/Figure.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+
+namespace Snake
+{
+    class Figure
+    {
+        public List<Point> pList;
```

Comment:
code convention violated
```suggestion
        public List<Point> PList;
```

but better
```suggestion
        public List<Point> PointsList;
```